packages/misc/commands/dm.js

```diff
@@ -47,37 +47,64 @@ module.exports = {
 			isSilent = interaction.options.getBoolean("silent");
 		}
 		*/
-		const msg = interaction.options.getString(
-			LANG.commands.dm.options.text.name,
-		);
 
 		const userId = interaction.options.getUser(
 			LANG.commands.dm.options.user.name,
 		);
-		const dmChannel = await userId.createDM();
+		if (userId === interaction.user.bot) {
+			await interaction.reply({
+				content: 'Botに対して実行することはできません!',
+				ephemeral: true,
+			});
+		}
```

Comment:
これ正常に動作しない
あと `userId` って変数名だけど実体は `User` だから `user` にしたほうがよさそう
```suggestion
		if (userId.bot) { // オプションのユーザーがBotであるか
			await interaction.reply({
				content: 'Botに対して実行することはできません!',
				ephemeral: true,
			});
            return; // 処理を終了
		}
```